common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -245,11 +245,13 @@ async function createSiblingsArray({
   toggles,
   workId, // id of current work being viewed
   openStatusOverride = false,
+  withChildren = true,
 }: {
   work: Work,
   toggles: Toggles,
   workId: string,
   openStatusOverride?: boolean,
+  withChildren?: boolean,
```

Comment:
If we set a default this isn't optional. As with the above property too.

Reply:
Is that true? I thought I'd expected that to be how it would be previously, but found Flow failed if the prop wasn't explicitly on the component. I might have imagined that, though.

Reply:
[I think you're right](https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoALgTwA4FMwAqYAvGAN4A+YCAhugFxgBGccMuNAdmBQL6pQArpwDG6AJZxu6ABRlqdEmCg0YAZ3y9GBAJTlUYMACdc6QUe610AblT8MchQ2WqNYXjodkPqIA)